### ISSUE TITLE
Integrating into an existing app

### ISSUE THREAD

**Issue by blocka** [hdorgeval/testcafe-starter#7]
It seems this repo is meant to use as a boilerplate for a greenfield app? Are there instructions for adding the infrastructure to an existing app?

**Comment by hdorgeval** [comment]
Hi @blocka,  this project is intended to be a start-from-scratch-template for your e2e testing. Just create new feature files in the features folder and adapt the `testcafe-config.ts` to fit your needs. If you think something is missing in the documentation do not hesitate to do a PR or to continue posting in this issue

**Comment by blocka** [comment]
Makes sense. But lots of folks already use their own boilerplates like cra
or vue- cli. Or perhaps they already have a brownfield app. It would be
nice to know what steps to take to add this into an existing codebase.

**Comment by hdorgeval** [comment]
Would be nice if you could share your code base, so that I can tell you what should be the steps to integrate that project.

**Comment by blocka** [comment]
Not gonna post one of my private projects here, but for the sake of example, let's take this: https://github.com/nicejade/awesome-vue-cli3-example

This is a project that was already bootstrapped with an existing tool (vue-cli)
